src/flux/media.coffee

```diff
@@ -2,6 +2,26 @@ _ = require 'underscore'
 htmlparser = require 'htmlparser2'
 {makeSimpleStore} = require './helpers'
 
+LINKS_BEGIN = ['#']
+LINKS_CONTAIN = ['/contents/', '.cnx.org']
```

Comment:
I was thinking 'cnx.org/contents/' would be good enough...i was trying to be more flexible and less strict...but yeah i guess there's no real reason to allow those

we don't try to get it from cnx directly, it tries to get if from the tutor-be